App.js

```diff
@@ -231,15 +231,16 @@ function ChatScreen({route, navigation}) {
   );
 }
 
-const Stack = isSmallScreenWidth ? createNativeStackNavigator() : createWebNavigator();
+const Stack = (isSmallScreenWidth && Platform.OS !== 'web') ? createNativeStackNavigator() : createWebNavigator();
```

Comment:
What about the iPad landscape case or any android/native platform in tablet size? I was thinking about using something like this, but the tablet usecase breaks it for me as in landscape we will need the createWebNavigator layout

Reply:
+1

Reply:
also I am not sure I agree that mobile web should use custom navigator. I would strongly encourage us to use the native stack navigator for this. Unless I am missing why we do not want to do that?

Reply:
> I would strongly encourage us to use the native stack navigator for this. Unless I am missing why we do not want to do that?

I agree, we should use it, but we need to figure out how to gracefully switch between them